src/erlcloud_ec2.erl

```diff
@@ -1348,37 +1374,66 @@ attribute_xpath(_, AttributeName) ->
     "/DescribeInstanceAttributeResponse/" ++ AttributeName ++ "/value".
 
 
+-type ec2_instances_ids() :: [string()].
+
 %%
+%% Function for making calls to DescribeInstances action
+%% DescribeInstances Documentation: docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeInstances.html
 %%
--spec describe_instances() -> {ok, [proplist()]} | {error, any()}.
-describe_instances() -> describe_instances([]).
+-spec describe_instances() -> ok_error(proplist()).
```

Comment:
This is changing from yielding a list of proplists to a single proplist. It looks like the original spec was incorrect and this is a fix.